ivy/functional/ivy/layers.py

```diff
@@ -1091,6 +1091,54 @@ def conv3d_transpose(
     )
 
 
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+def conv_transpose(
```

Comment:
I think rather than adding a `conv_transpose` function here, you should either add it in a separate PR along with their backend implementations.